packages/core/components/EditorPanel/FootnotesEditor.tsx

```diff
@@ -0,0 +1,77 @@
+import { UpdateFieldFunc } from '../../types/UpdateFieldFunc'
+import _ from 'lodash'
+import Footnotes, { Footnote } from '../../types/Footnotes'
+import { footnotesSymbols } from '../../helpers/footnoteSymbols'
+import InputSelect from '../inputs/InputSelect'
+import { TextField } from './Inputs'
+interface FootnotesEditorProps {
+  config: Footnotes
+  updateField: UpdateFieldFunc<Footnote[]>
+}
+
+const FootnotesEditor: React.FC<FootnotesEditorProps> = ({ config, updateField }) => {
+  const addStaticFootnote = () => {
+    const newStaticNotes = [...(config.staticFootnotes || []), { text: 'Add Footnote Text' }]
+    updateField(null, null, 'staticFootnotes', newStaticNotes)
+  }
+
+  const updateStaticFootnote = (footnoteIndex, footnoteUpdate: Footnote) => {
+    const footnoteCopy = _.cloneDeep(config.staticFootnotes)
+    footnoteCopy[footnoteIndex] = footnoteUpdate
+    updateField(null, null, 'staticFootnotes', footnoteCopy)
+  }
+
+  const deleteStaticFootnote = footnoteIndex => {
+    const footnoteCopy = _.cloneDeep(config.staticFootnotes)
+    footnoteCopy.splice(footnoteIndex, 1)
+    updateField(null, null, 'staticFootnotes', footnoteCopy)
+  }
+
+  const getOptions = (opts: string[]) => {
+    return [['', '--Select--']].concat(opts.map(key => [key, key]))
+  }
+
+  const datasets = config.datasets || {}
+
+  const dataColumns = config.dataKey ? getOptions(Object.keys(datasets[config.dataKey]?.data?.[0] || {})) : []
+  const dataSetOptions = getOptions(Object.keys(datasets))
+  return (
+    <>

```

Comment:
Implement `<Layout.Sidebar />` here so we have similar styles/visuals throughout the app.

Reply:
It looks like <Layout.Sidebar> is duplicating some of what EditorWrapper was doing. I've used <EditorWrapper> Around this. I think we need to do a separate refactor of EditorWrapper for consistency, and I don't want to put that in the scope of this PR.